database/migrations/2022_05_18_122013_add_location_geolocatie.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLocationGeolocatie extends Migration
+{
+    /**
+     * Run the migrations.
+     */
```

Comment:
```suggestion
```

Reply:
This is part of the Laravel template, it's literally used in all other migrations as well... I vote for consistency, and letting this stay is easier than removing all others :)